Gems/Atom/Feature/Common/Code/Include/Atom/Feature/CoreLights/DirectionalLightFeatureProcessorInterface.h

```diff
@@ -72,6 +72,9 @@ namespace AZ
             virtual void SetAngularDiameter(LightHandle handle, float angularDiameter) = 0;
 
             ////////// shadow specific
+            //! Sets wheter or not enabling shadow
+            virtual void SetShadowEnabled(LightHandle handle, bool enable) = 0;
```

Comment:
This can be passed as ref

Reply:
Do you mean passing the `LightHandle` by ref? Ah since all other parts pass it directly so I am following their current manner. Actually it is merely an `uint16_t`?

Reply:
I think it is a templated structure based on int16, but the remark is not a crucial one